### DISCUSSION TITLE
Add torrent from info hash

### DISCUSSION THREAD

**Discussion by joshhhhyyyy** [jesec/flood#520] [Ideas]
An option to add a torrent from the infohash. I tried doing this on my instance but it crashed and said unable to connect to transmission, so i presume this is not a feature thats implemented.

system info (if u need it): running flood on debian 11 bullseye with transmission. regular torrent magnet links work.

Please reference [This issue](https://github.com/transmission-remote-gui/transgui/issues/1090) for further context as the issue throughly summarises all my points.

**Comment by trim21** [comment]
https://github.com/jesec/flood/issues/736
